src/utils/FilterPeople.tsx

```diff
@@ -0,0 +1,65 @@
+import { useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+
+export const FilterPeople = (sourcePeople: Person[] | null) => {
+  const [searchParams] = useSearchParams();
+
+  if (sourcePeople === null) {
+    return null;
+  }
+
+  const sexFilter = searchParams.get('sex');
+  const query = searchParams.get('query');
+  const centuriesFilter = searchParams.getAll('centuries')
+    .map(century => (+century * 100));
+  const sortBy = searchParams.get('sort') as keyof Person;
```

Comment:
The helpers functions shouldn't know anything about search params, they should just filter/sort